src/developer/web-api/analytics.md

```diff
@@ -1846,6 +1846,188 @@ For example, to retrieve a list of enrollments from the "WHO RMNCH Tracker" prog
 
 The API supports using program indicators which are not associated to the "main" program (that is the program ID specified after `/query/`).
 
+## Dimensions { #webapi_dimensions }
+
+Four resources allow to easily retrieve data dimensions:
+
+- [Event Query data dimensions](#webapi_event_query_analytics_dimension)`/analytics/events/query/dimensions` 
+- [Event Aggregate data dimensions](#webapi_event_aggregate_analytics_dimension) `/analytics/events/aggregate/dimensions`
+- [Enrollment Query data dimensions](#webapi_enrollment_query_analytics_dimension) `/analytics/events/query/dimensions`
+- [Enrollment Aggregate data dimensions](#webapi_enrollment_aggregate_analytics_dimension) `/analytics/events/aggregate/dimensions`
+
+Resources mentioned above share the following request parameter:
+
+| Query parameter | required                                         | Description                        | Options |
+|-----------------|--------------------------------------------------|------------------------------------| --------|
+| filter          | no                                               | allows to specify output filtering.<br/>Format: `filter=field:OP:value;field:OP:value;...`| see [dimension filters section](#webapi_analytics_dimension_filters) |
+
```

Comment:
you could add other params to this table. Like: fields, order, page, pageSize

Reply:
I forgot fields. Paging are described in a separate section

Reply:
Yes, but should it be in a separate section? :) The table could describe all parameters that are common to those four endpoints. Otherwise, there's no need for a table with just the `filter` param IMO, as it is also described in its own section.